backend/Pipfile

```diff
@@ -0,0 +1,13 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+pylint = "*"
```

Comment:
`pylint` is a nice linter, but there is no definitive best linter for python. There is also a very popular `flake8` and for security `bandit` can be used (all 3 linters do things slightly different). I personally use all 3 in conjunction. Just FYI, sticking with only pylint is fine too.

Reply:
I've added both flake8 and bandit as dev-packages. I will setup Travis in another PR.